flight/goto/goto.py

```diff
@@ -0,0 +1,58 @@
+"""
+File containing the move_to function responsible
+for moving the drone to a certain waypoint
+"""
+import typing
+import asyncio
+from mavsdk import System
+import logging
+import math
+
+async def move_to(drone: System, latitude: float, longitude: float, altitude: float) -> None:
+    """
+    This function takes in a latitude, longitude and altitude and autonomously
+    moves the drone to that waypoint. This function will also auto convert the altitude
+    from feet to meters
+
+    Args:
+        drone (System): a drone object that has all offboard data needed for computation
+        latitude (float): a float containing the requested latittude to move to
+        longitude (float): a float containing the requested longitude to move to
+        altitude (float): a float contatining the requested altitude to go to (in meters)
+
+    Returns:
+        None
+    """
+    
+
+    #converts feet into meters
+    altitude = altitude * .3048
+
+    #get current altitude
+    async for terrain_info in drone.telemetry.home():
+        absolute_altitude: float = terrain_info.absolute_altitude_m
+        break
+
+    await drone.action.goto_location(latitude,longitude, altitude+absolute_altitude, 0)
+    location_reached: bool=False
+
+    #Loops until the waypoint is reached
+    while(not location_reached):
+        print("Going to waypoint")
```

Comment:
In the future, this will be added to a logging document to refer to the exact time when the drone begins to move to each object and when it arrives, so for now just comment the print statements out.